### DISCUSSION TITLE
Proposal for a new workflow on our kanban boards

### DISCUSSION THREAD

**Discussion by oliverschuerch** [swisspost/design-system#2971] [Ideas]
Our issues/tickets can have many states and it's not always clear what's the next step, after a ticket has been processed trough one of us.
To counteract this, I made a new Production and Design Board proposal:
- https://github.com/orgs/swisspost/projects/7/views/1
- https://github.com/orgs/swisspost/projects/7/views/4

There are a few more columns to provide more clarity on who is responsible for the next steps of the task.

Happy path:

| Column | Description | Workflow |
|-|-|-|
| Ready for work (backlog) | Issue has been created | Newly added issues and pull requests will end up here |
| Design in progress | Designer has assigned the ticket to himself and works on it | - |
| Design in review | Design changes done, needs review from a developer (developer can be assigned by designer or by himself) | - |
| Ready for development | Design ticket has been reviewed and needs development (remove assignements) | Repopened issues and pull requests with change requests will end up here |
| Develpment in progress | Developer has assigned the ticket to himself and works on it | - |
| Develpment in code review | Code changes done, needs review form a developer (developer can be assinged by github or manually) | - |
| Development in design review | Code review done, needs review from a designer (designer updates figma state when approved) (designer can be assigned by developer or by himself) | Approved pull requests will end up here |
| Done | All reviews have been done, developer merged the ticket | Closed issues and merged pull requests will end up here |

Development only path:

| Column | Description |
|-|-|
| Ready for work (backlog) | Issue has been created |
| Develpment in progress | Developer has assigned the ticket to himself and works on it |
| Develpment in code review | Code changes done, needs review form a developer |
| Development in design review | Code review done, needs review from a designer (optional) |
| Done | All reviews have been done, developer merged the ticket |

Design only path:

| Column | Description |
|-|-|
| Ready for work (backlog) | Issue has been created |
| Design in progress | Designer has assigned the ticket to himself and works on it |
| Design in review | Design changes done, needs review from a developer |
| Done | All reviews have been done, designer closes the ticket |

As the name suggests, this is just a proposal.
Please add your opinion and discuss whether we try this out, change something, stay with the current Kanban board or whatever...

**Comment by rouvenpost** [comment]
I would recommend to change «Done» in to «Ready to use».
